ecs-cli/modules/cli/image/image_app.go

```diff
@@ -68,8 +73,11 @@ func ImagePush(c *cli.Context) {
 
 	ecrClient := getECRClient(c, commandConfig)
 	stsClient := stsclient.NewClient(commandConfig)
+	taggingClient := tagging.NewTaggingClient(commandConfig)
```

Comment:
Does the tagging clieng have FIPS endpoints? If so might have to do something similar as what we do in `getECRClient`.

Reply:
That is true for now, though the idea is that we can use this flag for other services as well given that FIPS endpoints now exist in ECS and other services.